pubspec.lock

```diff
@@ -511,6 +511,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.8.2"
+  stack:
```

Comment:
for most of the cases I think it's not recommended for us to push any changes from `pubspec.lock` since this is generated after we performed `flutter pub get` based on your local machine's current config, unless @raelsj will require to for any important reason I would say 🤔

Reply:
should I add this to .gitignore?